draft-ietf-rats-corim.md

```diff
@@ -1429,9 +1429,155 @@ is extended using Endorsements etc. from the accepted tags.
 
 ### Matching Evidence against Reference Values
 
+This section describes the process performed by the Verifier to determine
+which Endoresments in the candidate CoMIDs are applicable to the Attester being
+verified. All applicable Endorsements are added to the Accepted Claims Set.
+
+The verifier checks whether Endorsements are applicable by comparing Evidence
+in the Accepted Claims Set against Reference Values from the CoMID. These
+Reference Values may be provided as Reference Value Triples or may be combined
+with the Endorsements, for example as the Conditional Endorsement Series Triple.
+
+The following subsections describe how the CoRIM tells the verifier which
+Reference Values and Endorsements are grouped together ({{sec-grouping-ref-vals}})
+and how the verifier matches a Reference Value against the Accepted Claims Set
+({{sec-match-one-ref-val}}).
+
+If all Reference Values in a group match the Accepted Claims Set then all
+Endorsements in the group are added to the Accepted Claims Set
+(see {{sec-add-to-acs}}).
+
+If any Reference Value in a group does not match then this does not affect
+the processing of other groups.
+
```

Comment:
Here I would like to mention one statement:
`When a Reference Value mismatch occurs, its impact on overall Appraisal  depends on the Verifier Appraisal Policy used in the context of the Appraisal.`

Reply:
I disagree with this change. The point of CoRIM is for the Endorser (rim author) to provide a description of what they want. If the Verifier ignores the mismatch then it is not following the instructions from the Endorser.
I will find a way to say this which is hopefully acceptable.

Reply:
Well... a CoRIM does not convey policy (only conditional-endorsements as a fringe case, I think). Therefore, there is some discrepancy here. It must be very simple and clear, though, for a Verifier to select appropriate policies or fall-back to a default behavior, I think.

Reply:
Consider the Attester being a graph where each node represents a TE. Appraisal of each TE will result in an isomorphic graph containing accepted claims for each TE. Some of the nodes may have an appraisal failure result.
What is the expected behavior of the Verifier when preparing an AR? One possibility is there is an AR graph that is isomorphic to the evidence / accepted claims graphs. The concise attestation results (CAR) schema would allow each node in the graph to have an AR4SI result while the EAR schema would take the average of the graph. Presumably, the nodes containing failed appraisal results would essentially multiply the average by zero causing the whole result to be "failure"? Since the exact behavior of how Verifier produces attestation results isn't known, the verifier should maintain the integrity of the (accepted claims) graph and treat it as the input to the verifier's AR processing capability.
Possibly, it makes sense to add a section following section 5 that describes attestation results processing?

Reply:
to be addressed with issue #136